File: block29-Spring-Reactive-cuatroenraya/Backend-cuatroenraya/src/main/java/com/example/block29SpringReactivecuatroenraya/session/entity/Metadata.java
package com.example.block29SpringReactivecuatroenraya.session.entity;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class Metadata {

    private int sessionId;

    private String playerId;

    private Integer column;

    private Integer round;
}

File: block29-Spring-Reactive-cuatroenraya/Frontend-cuatroenraya/src/app/metadata.ts
export class Metadata {
    
    sessionId: string;
    playerId: string;
    column: number;
    round: number;

    constructor(sessionId: string, playerId: string, column: number, round: number){
            this.sessionId = sessionId;
            this.playerId = playerId;
            this.column = column;
            this.round = round;
    }
}